<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora de Pre√ßos (Aula 16 - CheckBox)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(85, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>206</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;font-weight:bold;color:#fff;</string>
    </property>
    <property name="text">
     <string>Valor total:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>R$ 1.500,00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>R$ 20,00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>R$ 10,00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>R$ 32,00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>R$ 5,50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:25px;font-weight:bold;color:rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>R$ 0,00</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
